app/controllers/courses_controller.rb

```diff
@@ -0,0 +1,38 @@
+class CoursesController < ApplicationController
+
+  def new
+    @course = Course.new
+  end
+
+  def create
+    @course = Course.new(course_params)
+    if @course.save
+      flash[:success] = "Course successfully registered"
+      redirect_to current_user
```

Comment:
should be redirect to course detail or list of courses
```suggestion
      redirect_to courses_url
```